cluster/addons/cluster-monitoring/google/heapster-controller.yaml

```diff
@@ -1,30 +1,30 @@
 apiVersion: v1
 kind: ReplicationController
 metadata: 
-  name: monitoring-heapster-v2
+  name: monitoring-heapster-v3
   namespace: default
   labels: 
     k8s-app: heapster
-    version: v2
+    version: v3
     kubernetes.io/cluster-service: "true"
 spec: 
   replicas: 1
   selector: 
     k8s-app: heapster
-    version: v2
+    version: v3
   template: 
     metadata: 
       labels: 
         k8s-app: heapster
-        version: v2
+        version: v3
         kubernetes.io/cluster-service: "true"
     spec: 
       containers: 
-        - image: gcr.io/google_containers/heapster:v0.14.0
+        - image: gcr.io/google_containers/heapster:v0.14.1
           name: heapster
           command:
             - /heapster
-            - --source=kubernetes:https://kubernetes
```

Comment:
API server url isn't required. Heapster will pick up from the configs in the token.